test/check-machines-settings

```diff
@@ -705,6 +710,28 @@ class TestMachinesSettings(machineslib.VirtualMachinesCase):
         b.wait_visible("#watchdog-dialog-apply[aria-disabled=true]")
         b.mouse("#watchdog-dialog-apply", "mouseenter")
         b.wait_visible("#watchdog-live-edit-tooltip")
+        b.click("#vm-subVmTest2-watchdog-modal button[aria-label=Close]")
+        b.wait_not_present("#vm-subVmTest2-watchdog-modal")
+
+        self.goToMainPage()
+        self.waitPageInit()
+
+        m.execute("virsh destroy subVmTest2")
+
+        # Create a Q35 machine, which has a permanent watchdog that
+        # can't be removed.  (Some older versions of libvirt don't add
+        # a watchdog.)
+
+        m.execute("virsh destroy subVmTest1; virsh undefine subVmTest1")
+        args = self.createVm("subVmTest1", os="linux2022")
+        has_watchdog = (m.execute("virsh dumpxml subVmTest1 | grep '<watchdog' || echo no") != "no\n")
```

Comment:
This is a strange way to write that, and it also hides errors in the pipeline. How about `"<watchdog" in m.execute("virsh..")`?

But moreover, this is too dynamic for a test for my taste -- if the above comment is true, then this should always be `True`, no? So this could instead be an assertion? Or at least a more static `if m.image in [old, OSes]`?

Reply:
Agreed!

Reply:
Is [this failure](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1600-e72e108e-20240503-103655-rhel-8-10/log.html#87) related? Sounds plausible to put RHEL 8 into the "old" bucket.